server/db/init.js

```diff
@@ -19,10 +19,119 @@ bookshelf.knex.schema.hasTable('users').then((exist) => {
+      table.enu('status', ['draft', 'active', 'pause', 'completed']).notNullable();
+      table.integer('script_id').references('scripts.id').notNullable();
+      table.timestamp('created_at').defaultTo(bookshelf.knex.fn.now());
+      table.timestamp('updated_at').defaultTo(bookshelf.knex.fn.now());
+    }).then(() => {
+      console.log(('Created campaigns table'));
+    }).catch(err => console.log('Error creating campaigns table', err));
+  }
+});
+
+bookshelf.knex.schema.hasTable('scripts_to_questions').then((exist) => {
+  if (!exist) {
+    bookshelf.knex.schema.createTable('scripts_to_questions', (table) => {
+      table.increments().primary();
+      table.integer('script_id').references('scripts.id').notNullable();
+      table.integer('question_id').references('questions.id').notNullable();
+      table.integer('sequence_number').notNullable();
+      table.timestamp('created_at').defaultTo(bookshelf.knex.fn.now());
+      table.timestamp('updated_at').defaultTo(bookshelf.knex.fn.now());
+    }).then(() => {
+      console.log('Created scripts_to_questions table');
+    }).catch(err => console.log('Error creating scripts_to_questions table', err));
+  }
+});
+
+bookshelf.knex.schema.hasTable('contacts').then((exist) => {
+  if (!exist) {
+    bookshelf.knex.schema.createTable('contacts', (table) => {
+      table.increments('id').primary();
+      table.string('external_id').nullable();
+      table.string('first_name').notNullable();
+      table.string('last_name').nullable();
+      table.string('email').nullable();
+      table.string('phone_number').notNullable().unique().index();
+      table.boolean('is_invalid_number').defaultTo(false);
+      table.boolean('do_not_call').defaultTo(false);
+      table.timestamp('created_at').defaultTo(bookshelf.knex.fn.now());
+      table.timestamp('updated_at').defaultTo(bookshelf.knex.fn.now());
+    }).then(() => {
+      console.log(('Created contacts table'));
+    }).catch(err => console.log('Error creating contacts table', err));
+  }
+});
+
+bookshelf.knex.schema.hasTable('contact_lists').then((exist) => {
+  if (!exist) {
+    bookshelf.knex.schema.createTable('contact_lists', (table) => {
+      table.increments('id').primary();
+      table.string('name').notNullable().unique();
+      table.timestamp('created_at').defaultTo(bookshelf.knex.fn.now());
+      table.timestamp('updated_at').defaultTo(bookshelf.knex.fn.now());
+    }).then(() => {
+      console.log(('Created contact_lists table'));
+    }).catch(err => console.log('Error creating contact_lists table', err));
+  }
+});
+
+bookshelf.knex.schema.hasTable('contacts_contact_lists').then((exist) => {
+  if (!exist) {
+    bookshelf.knex.schema.createTable('contacts_contact_lists', (table) => {
+      table.increments('id').primary();
+      table.integer('contact_id').references('contacts.id').notNullable();
+      table.integer('contact_list_id').references('contact_lists.id').notNullable();
```

Comment:
@perfectlynormalbeast we added it in, thanks for catching that! it makes sense we should check uniqueness in that case